app/assets/stylesheets/forever_style_guide/modules/_hero_simple.scss

```diff
@@ -0,0 +1,89 @@
+.hero_simple {
+  position: relative;
+  height: 500px;
+  background-size: cover;
+
+  h1, p {
+    color: color('white');
```

Comment:
Also, from a design perspective I think we might want to bump up the text size just a bit on the subheader.  And from a markup standpoint it's probably more valuable from a semantics and search standpoint for the subheader to be an h2 rather than a paragraph.


![screen shot 2016-04-14 at 10 19 32 am](https://cloud.githubusercontent.com/assets/3814998/14531247/8474087c-022a-11e6-8f77-06d0051e7662.png)